ceph-releases/luminous/ubuntu/16.04/daemon/disk_introspection.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+HOSTNAME=$(uname -n)
+MY_NAMESPACE=$(kubectl get pods "$HOSTNAME" -o template --template="{{.metadata.namespace}}")
+DISK_FILE=$(kubectl --namespace="$MY_NAMESPACE" get pods "$HOSTNAME" -o template --template="{{.spec.nodeName}}")-disks
```

Comment:
If we make the change above, let's replace this with a jsonpath template for consistency:

```
DISK_FILE=$(kubectl --namespace="${MY_NAMESPACE}" get pods "${HOSTNAME}" -o jsonpath="{.spec.nodeName}")-disks
```

Alternatively, if we also want to query all namespaces for this pod's node name we can use this command:

```
DISK_FILE=$(kubectl get pods --all-namespaces -o jsonpath="{.items[?(@.metadata.name == \"${HOSTNAME}\")].spec.nodeName}")-disks
```